packages/openci-controller/Makefile

```
@@ -1,4 +1,10 @@
-.PHONY: down up-dev full-up-dev up-prod ensure-env build rebuild
+# Load environment variables from .env file if it exists
+ifneq ($(wildcard ./.env),)
+	include .env
+	export
+endif
+
+.PHONY: down up-dev full-up-dev up-prod ensure-env build rebuild db-schema
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)